features/sharding/core/src/main/java/org/apache/shardingsphere/sharding/yaml/engine/representer/processor/NoneYamlTupleProcessor.java

```diff
@@ -37,11 +37,11 @@ public String getTupleName() {
     
     @Override
     public NodeTuple process(final NodeTuple nodeTuple) {
-        return (isNullNode(nodeTuple.getValueNode()) || isEmptyMappingNode(nodeTuple.getValueNode())) ? null : processNoneTuple(nodeTuple);
+        return (isNullNode(nodeTuple.getKeyNode(), nodeTuple.getValueNode()) || isEmptyMappingNode(nodeTuple.getValueNode())) ? processNoneTuple(nodeTuple) : null;
```

Comment:
Since `processNoneTuple` invocation is reversed with `null`, could we add more unit test for cases fit `isNullNode` or `isEmptyMappingNode`?

Reply:
Hi, I just made some changes to this line. Could you please have a look? Does it make more sense?

Reply:
So is the issue caused by `isEmptyMappingNode`?